admin-ui/README.md

```diff
@@ -1,27 +1,65 @@
 # Gluu Admin UI
 
-This project has been built on [Airframe React](https://github.com/0wczar/airframe-react). 
+The Gluu Flex Admin UI is a reactive web interface to simplify the management and configuration of your Auth Server. The Admin UI enables you to easily view and edit configuration properties, interception scripts, clients, and metrics in one place.
 
-## Installation 
+## Introduction
+
+This document outlines the steps to setup the project for development, including installation, configuration, and running the application.
 
 ### Prerequisites
 
-Node, NPM (latest, stable version)
+Before setting up the project, ensure you have the following installed :
+
+- NodeJS : v18.xx.x or above
+- NPM : 8.xx or above
+- Java : openjdk 17.x.x or above
+
+### Installation
+
+Follow these steps to setup the project :
 
-### Development
+1. Clone the project
+
+```
+git clone https://github.com/GluuFederation/flex
+cd flex/admin-ui
+```
 
-To run the project locally execute following commands.
+2. Create a file name `.env.development.local` with following contents.
+
+```
+BASE_PATH=/admin/
```

Comment:
`https://admin-ui-test.gluu.org` is the test-server we are currently using for local development in Gluu's Admin UI team. I think the purpose of adding README is people outside Gluu and set-up and run Admin UI project locally (since Admin UI is ). But they should not use `admin-ui-test.gluu.org`. 

So my opinion is people should install Flex on vm and add its details in `.env.development.local`.